keyboards/crkbd/keymaps/mcrown/rules.mk

```diff
@@ -0,0 +1,28 @@
+
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE = no        # Mouse keys(+4700)
+EXTRAKEY_ENABLE = no        # Audio control and System control(+450)
+CONSOLE_ENABLE = no         # Console for debug(+400)
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = no            # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+MIDI_ENABLE = no            # MIDI controls
+AUDIO_ENABLE = no           # Audio output on port C6
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGB_MATRIX_ENABLE = yes     # Enable RGB Matrix. 
+SWAP_HANDS_ENABLE = no      # Enable one-hand typing
+OLED_DRIVER_ENABLE = yes
+
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = yes    # Breathing sleep LED during USB suspend
```

Comment:
```suggestion
MOUSEKEY_ENABLE = no
SLEEP_LED_ENABLE = yes
RGB_MATRIX_ENABLE = yes
OLED_DRIVER_ENABLE = yes
```

Reply:
Hello @fauxpark why does this change is needed?

Reply:
The keyboard- and keymap-level rules.mk files are combined together during compilation, so you only need to specify the settings you want to change.